main.go

```diff
@@ -599,6 +634,12 @@ func convertWindupWhenToAnalyzer(windupWhen windup.When, where map[string]string
 			pattern := strings.Replace(substituteWhere(where, jc.References), "{*}", "*", -1)
 			pattern = strings.Replace(pattern, "(*)", "*", -1)
 			pattern = strings.Replace(pattern, ".*", "*", -1)
+			// Make some .* regesx's more generic.
```

Comment:
hmm, some these change the semantic value of the regex pretty significantly, I'm not sure we want to be doing that

Reply:
Discussed offline, 

we are choosing to keep this and then to make a todo to come back and deal with capture rules.